daml-script/export/src/main/scala/com/daml/script/export/TreeUtils.scala

```diff
@@ -194,12 +194,15 @@ object TreeUtils {
   def treeEventCreatedConsumedCids(
       event: TreeEvent.Kind,
       tree: TransactionTree,
-  ): (Set[ContractId], Set[ContractId]) = {
-    var created = Set.empty[ContractId]
+  ): (Seq[SimpleCreatedContract], Set[ContractId]) = {
+    var created = Seq.empty[SimpleCreatedContract]
```

Comment:
Do not use Seq, but prefer a concrete instance of `Seq`, otherwise the complexity of your operation are not explicit. 
Some argues that input argument should be the most generic and output the most concrete (following covariance/variance or input/output), but I will argue, except if you design a general library, always use a concrete type to make the cost of your operation explicit. 
In this case `List` is probably a good choice.

Reply:
thanks, fix commit coming up